index.html

```diff
@@ -4805,8 +4805,64 @@ <h2>
 layers, and pad messages to standard lengths.
       </p>
     </section>
-  </section>
 
+    <section>
+      <h2>
+          Service Privacy
+      </h2>
+      <p>
+The ability for a controller to optionally state at least one service endpoint
+in the DID document increases their control and agency. Each additional
+endpoint in the DID document adds privacy risk either due to correlation
+(e.g., across endpoint descriptions) or because the services are not
+protected by an authorization mechanism, or both.
+      </p>
+      <p>
+The degree of additional privacy risk caused by using multiple service endpoints in one
+DID document can be difficult to estimate. Privacy harms are typically
+unintended consequences. DIDs can identify documents, services, schemas, and
+other things that may be associated with individual people,
+households, clubs, and employers &mdash; and correlation of their service
+endpoints could become a powerful surveillance and inference tool.
```

Comment:
The degree of additional privacy risk caused by using multiple service 
endpoints in one DID document can be difficult to estimate. Privacy 
harms are typically unintended consequences. DIDs can identify documents, 
services, schemas, and other things that may be associated 
with individual people, households, clubs, and employers &mdash; and 
correlation of their service endpoints could become a powerful 
surveillance and inference tool.